native/effektio/src/api/client.rs

```diff
@@ -47,6 +47,8 @@ impl std::ops::Deref for Client {
     }
 }
 
+static mut CURRENT_CLIENT: Option<MatrixClient> = None;
```

Comment:
we can't do that, sorry. The client is hold by the surrounding SDK and the code may not assume that there is a global to store the current client. Why? Because the SDK is meant to allow for multi-client support (even if the UI doesn't implement it yet).

Reply:
Yes, correct.
It doesn't look smart for me too.
But I guess on rust side there is no way to manage the multiple clients and set active to some client.
Alternatively on flutter side there is `currentClient` of `EffektioSdk`.
If we go and back between rust and flutter, we can get delay.
Before flutter can deliver current client to rust, flutter must pass token to rust and rust must login with it, so it takes a bit of time to perform this login, because login is operation via network.
Really I tested it and confirmed delay when opening `ChatScreen` page.
So I just wanted to finish all operations in rust, without go and back between rust and flutter.

Okay.
Will work from `currentClient` of `EffektioSdk` of flutter side.

Reply:
Did you tested it in '''release''' build?

Reply:
No, I tested in debug build